internal/pkg/deploy/cloudformation/stack.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cloudformation
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+)
+
+// DeleteStack calls the CloudFormation DeleteStack API with the input stack name.
+func (cf CloudFormation) DeleteStack(stackName string) error {
```

Comment:
I think these should be private methods that `DeleteApp`/`DeleteEnv` can use :)

Reply:
These are simple wrappers around the CloudFormation API, so I prefer to leave them public so that someone can take advantage of them if they need them.